static/benchmarks/css/competition.sass

```diff
@@ -7,7 +8,10 @@
     border: 1px solid $brain-score-color
 
 .competition-hero
-  background-color: $brain-score-color
+
+  background: rgb(0,0,0)
```

Comment:
I'm guessing this first background specification is for browsers not supporting linear-gradient? Shouldn't that be `background: $brain-score-color` then?